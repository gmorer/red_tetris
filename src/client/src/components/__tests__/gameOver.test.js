import GameOver from './../gameOver'
import React from 'react';
import renderer from 'react-test-renderer';

test('Some test', () => {
	const component = renderer.create(<GameOver style={{}} />)
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})