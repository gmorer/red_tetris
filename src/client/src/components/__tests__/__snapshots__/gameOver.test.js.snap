// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some test 1`] = `
<div
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0,  0.5)",
      "height": "100%",
      "position": "absolute",
      "textAlign": "center",
      "width": "100%",
      "zIndex": "99",
    }
  }
>
  <h3
    style={
      Object {
        "color": "red",
        "fontSize": "3em",
        "marginTop": "50%",
      }
    }
  >
    Game
    <br />
    Over
  </h3>
</div>
`;
